Update on index.tmpl

gusman opened the pull request:
- Fix #32, update the code as mentioned in issue description
- Fix #33, update thy cyto template by removing ${type} prefix
- Fix #34, by changing the approach for executing ssh command in clab server (host) to generate ssh command that can be copied by user on his terminal.

asadarafat committed:
Update docker-ci-v2.yml

gusman committed:
fix #32

gusman committed:
Add sudo and containerlab fullpath when executing
clab netem command

gusman committed:
fix #33

gusman committed:
Fix #34
Change approach from executing command in clab servers
to generate ssh command which can be copied on user terminal

gusman committed:
Update docker-ci-v3.yaml to support
tag extraction from git tag reference

asadarafat commented on .github/workflows/docker-ci-v2.yml:
dont change this. keep it as reference. ci-v3 will be used

gusman replied:
I have changed back `docker-ci-v2.yaml` to be same as `master`/`main` branch

asadarafat commented on .github/workflows/docker-ci-v3.yml:
how `type=ref,event=tag w` will be used?

gusman replied:
By adding `type=ref,event=tag`, it allows me to build and publish the docker image by selecting `git tag` value,
`git tag` become the tag of docker image as well.

asadarafat commented on dist/html-static/clab-client/clab-client-mac/clab-capture.scpt:
let's keep this file for reference

gusman replied:
I rebase the PR code to excluded changes in `./dist` directory

asadarafat reviewed (changes requested):
please change the request after addressing the comments i gave.

gusman committed:
Change github worflow docker-ci-v2.yaml to be
same as master/main branch

gusman commented:
> please change the request after addressing the comments i gave.

I have addressed all the comments.
Need you review again.
If the all the changes are OK, please help to build and publish new docker image.
